seed/building_sync/validation_client.py

```diff
@@ -13,16 +13,11 @@ class ValidationClientException(Exception):
 
 
 def _validation_api_post(file_, schema_version, use_case_name):
-    payload = {'schema_version': schema_version}
-    files = [
-        ('file', file_)
-    ]
-
     return requests.request(
         "POST",
         VALIDATION_API_URL,
-        data=payload,
-        files=files,
+        data={'schema_version': schema_version},
```

Comment:
Ugh, I just found out that if someone uploads a v2.0 file, we will pass a version of 2.0 (which is correct according to our tags) but selection tool requires 2.0.0. Can we add a check in this function to change version to 2.0.0 if someone passes BUILDINGSYNC_V2_0?

Reply:
I instead just updated BUILDINGSYNC_V2_0 to be "2.0.0" as I don't believe it's ever used to verify "2.0".  Can you see any problems with this?

Reply:
I'm worried that might cause some issues, our official tag is [v2.0](https://github.com/BuildingSync/schema/releases/download/v2.0/BuildingSync.xsd) (not 2.0.0) which people would use in schemaLocation to reference the schema, which we were parsing out to determine the BSync version originally (before v2.1 added the `@version` attribute).